src/views/dialogs/HelpDialog.tsx

```diff
@@ -74,7 +75,13 @@ export const HelpDialog = ({ setIsOpen }: ElementProps) => {
       slotFooter={
         latestCommit ? (
           <Styled.Footer>
-            Release - <span title={latestCommit}>{`${latestCommit.substring(0, 7)}`}</span>
+            <span>
+              Release - <span title={latestCommit}> {`${latestCommit.substring(0, 7)}`}</span>
+            </span>
+            <span>
```

Comment:
lets hide it if `latestVersion` is undefined. same way `latestCommit` is checked